src/Core/Amqp/RabbitMqQueueLister.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+/**
+ * This File is part of JTL-Software
+ *
+ * User: marius
+ * Date: 7/27/21
+ */
+
+namespace JTL\SCX\Lib\Channel\Core\Amqp;
+
+use JsonException;
+use JTL\Nachricht\Contract\Transport\Amqp\AmqpQueueLister;
+use JTL\Nachricht\Transport\Amqp\AmqpHttpConnectionFailedException;
+use JTL\Nachricht\Transport\Amqp\AmqpTransport;
+
+class RabbitMqQueueLister implements AmqpQueueLister
+{
+    private AmqpTransport $transport;
+
+    public function __construct(AmqpTransport $transport)
+    {
+        $this->transport = $transport;
+    }
+
+    /**
+     * @throws AmqpHttpConnectionFailedException
+     * @throws JsonException
+     */
+    public function listQueues(string $queuePrefix = null): array
```

Comment:
Wie sehen die Daten aus die hier zurück kommen? 

Nur ein string dann pack die Beschreibung der Array Struktur als docblock in den Header
````
@return array<int, string>
````